src/main/java/com/gmail/goosius/siegewar/enums/SiegeStatus.java

```diff
@@ -72,6 +72,35 @@ public String getName() {
 		return this.name;
 	}
 
+	public boolean allowsPlundering() {
+		return attackersWon();
+	}
+
+	public boolean allowsInvading() {
+		return attackersWon();
+	}
+
+	public boolean attackersWon() {
+		switch(this) {
+		case ATTACKER_WIN:
+		case ATTACKER_CLOSE_WIN:
+		case DEFENDER_SURRENDER:
+			return true;
+		default:
+			return false;
+		}
+	}
+
+	public boolean defendersWon() {
+		switch(this) {
+		case DEFENDER_WIN:
```

Comment:
🚧 
DEFENDER_CLOSE_WIN presumably should be here too

Reply:
I was just basing this off of the code that this replaces:

```java
if (siege.getStatus() == SiegeStatus.DEFENDER_WIN || siege.getStatus() == SiegeStatus.ATTACKER_ABANDON)
```

Should it also have been accounted for in PlunderTown?
```java
		if(siege.isRevoltSiege()) {
			// If the rebels won, plunder is not possible
			if (siege.getStatus() == SiegeStatus.DEFENDER_WIN || siege.getStatus() == SiegeStatus.ATTACKER_ABANDON)
				throw new TownyException(translator.of("msg_err_siege_war_plunder_not_possible_rebels_won"));
		}
```

Reply:
That's right. That's a bug. I'll fix.